plenum/server/domain_req_handler.py

```diff
@@ -175,48 +175,9 @@ def get_value_from_state(self, path, head_hash=None, with_proof=False):
         :param get_value: whether to return the value
         :return: a state proof or None
         '''
-        root_hash = head_hash if head_hash else self.state.committedHeadHash
-        encoded_root_hash = state_roots_serializer.serialize(bytes(root_hash))
-
-        if not with_proof:
-            return self.state.get_for_root_hash(root_hash, path), None
-
-        multi_sig = self.bls_store.get(encoded_root_hash)
         if not multi_sig:
```

Comment:
It looks like multi-sig will be calculated even if it's not needed, that is if `with_proof=False`